packages/eslint-plugin-pf-codemods/lib/rules/v5/table-rename-TableComposable.js

```diff
@@ -0,0 +1,91 @@
+const {getPackageImports} = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8892
+module.exports = {
+  meta: {fixable: "code"},
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-table");
+    const tableComposableImport = imports.find(
+      (imp) => imp.imported.name === "TableComposable"
+    );
+
+    return imports.length === 0 || !tableComposableImport
+      ? {}
+      : {
+          // update opening/closing elements
+          JSXIdentifier(node) {
+            const nodeName = node.name;
+            const importedNode = imports.find(
+              (imp) => imp.local.name === nodeName
+            );
+            const isAliased =
+              importedNode?.local?.name !== importedNode?.imported?.name;
+            if (nodeName === "Table") {
+              // if data-codemods attribute, do nothing
+              const parentNode = node.parent;
+              const isOpeningTag = parentNode.type === "JSXOpeningElement";
+              const openingTagAttributes = isOpeningTag
+                ? parentNode.attributes
+                : parentNode.parent.openingElement.attributes;
+              const hasDataAttr =
+                openingTagAttributes &&
+                openingTagAttributes.filter(
+                  (attr) => attr.name?.name === "data-codemods"
+                ).length;
+              if (hasDataAttr) {
+                return;
+              }
+            }
+            if (nodeName === "TableComposable" || isAliased) {
```

Comment:
wouldn't this catch on any node that is using an alias import as long as a tableComposable exists?

i think we'd need to check that `nodeName === tableComposableImport.local.name`

Reply:
Yes nice catch, thanks